packages/contracts/contracts/interpreters/FulfillInterpreter.sol

```diff
@@ -98,8 +99,9 @@ contract FulfillInterpreter is ReentrancyGuard, IFulfillInterpreter {
       amount,
       callData,
       returnData,
-      success
+      success,
+      isContract
     );
-    return (success, returnData);
+    return (success, isContract, returnData);
```

Comment:
personally I would have preferred to tack new variables on to the end of the return tuple here, but is there a reason it's going in the middle?

Reply:
they're both booleans, so keeping the same data types next to each other

Reply:
fair 100%